game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/BattleSteps.java

```diff
@@ -185,20 +189,14 @@ public boolean isAmphibious() {
       steps.addAll(offensiveFirstStrike.getNames());
     }
     steps.addAll(airAttackVsNonSubs.getNames());
+    steps.addAll(offensiveStandard.getNames());
 
-    if (attackingUnits.stream().anyMatch(Matches.unitIsFirstStrike().negate())) {
-      steps.add(attacker.getName() + FIRE);
```

Comment:
Some elements have multiple steps, eg: ` steps.addAll(submergeSubsVsOnlyAir.getNames());`, can we use the same pattern here for `attacker`? EG: `attacker.getNames()`? This way we can avoid forming the 'fire & select casualties' here.